react/Summary.tsx

```diff
@@ -36,10 +36,12 @@ const Summary: FunctionComponent<SummaryProps> = ({
 }) => {
   return (
     <SummaryContext.Provider value={{ totalizers: totalizers, total: total }}>
-      <h5 className="t-heading-5 mt6 mb5">
-        <FormattedMessage id="store/checkout-summary.Summary" />
-      </h5>
-      <div className="c-on-base">{children}</div>
+      <div>
+        <h5 className="t-heading-5 mt0 mb5">
```

Comment:
Do we need `mt0` and `mb5` here? Shouldn't those be applied via `flex-layout`?

Reply:
`h5` has top and bottom margins by default. So i'm removing that marginTop and setting a marginBottom according to Figma layout.

I don't think that `h5` needs an exclusive block. Do you?

Reply:
I'm dumb, I thought these margins were being applied to the whole component. This seems fine to me as it is now.